fnproject/fn
# Add API to configure runner

We need the ability to configure a remote runner. The data we pass is implementation specific, so we keep it generic enough. The pure_runner implementation dumps the data into a file. The file path is configurable.

- Link to issue this resolves

- What I did
Added a new rpc ConfigureRunner to RunnerProtocolServer.

- How I did it
Updated the runner.proto file with new rpc.

- How to verify it
Make sure the build succeeds

- One line description for the changelog
Add ConfigureRunner rpc to RunnerProtocolServer.

- One moving picture involving robots (not mandatory but encouraged)
